examples/unconstrained-map-warning/index.mjs

```diff
@@ -0,0 +1 @@
+// index.rsh is not used in this example
```

Comment:
Typo

Reply:
But also, this should just be in hs/t/y if there's no runtime component

Reply:
I also figured it might as well be a test at this point.